pkg/network/ebpf/c/tracer.h

```diff
@@ -29,9 +31,10 @@ typedef struct {
     __u64 timestamp;
     __u32 flags;
     __u32 cookie;
-    __u8 direction;
     __u64 sent_packets;
     __u64 recv_packets;
+    __u8 direction;
+    protocol_t protocol;
```

Comment:
Should this be (small) array instead? Like `protocol_t protocol[3];`? Or is this for the interim, as we add more support later?

Reply:
The for the interim
In the future ill change it to an array